query-service-impl/build.gradle.kts

```diff
@@ -13,16 +13,16 @@ dependencies {
     implementation("io.netty:netty:3.10.6.Final") {
       because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-30430")
     }
-    implementation("io.netty:netty-common:4.1.77.Final") {
+    implementation("io.netty:netty-common:4.1.94.Final") {
       because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-2812456")
     }
     implementation("org.apache.zookeeper:zookeeper:3.6.3") {
       because("Multiple vulnerabilities")
     }
-    implementation("io.netty:netty-transport-native-epoll:4.1.71.Final") {
+    implementation("io.netty:netty-transport-native-epoll:4.1.94.Final") {
```

Comment:
netty should be upgraded downstream (grpc utils), can try removing these 4. MAy need to maintain that 3.x one.

Reply:
Removed them